client/modules/api/mockoon/mockoon-api.json

```diff
@@ -0,0 +1,12235 @@
+{
```

Comment:
This file is huge, was it autogenerated?

Reply:
Sorta - you can treat it like a binary when added all at once like this. It's Mockoon's exported "environment" for Wikijump's API, which was derived from the OpenAPI schema. Smaller changes to the environment should actually be fairly readable.